src/lib/components/textarea/textarea.js

```diff
@@ -46,6 +46,11 @@ class Textarea extends PureComponent {
       autoFocus,
       isValidationOk,
       renderValidationError,
+      validationMessage,
+      hasErrorAfterSubmit,
+      getValidationMessage,
+      onChangeFunction,
+      handleValidations,
```

Comment:
idem here

Reply:
aaand same – sorry, should have explained why I am doing it in a comment